posts/TechYesWeCan_Pt1.md

```diff
@@ -0,0 +1,115 @@
+
+It was advised, and properly so, that the ramblings of six women and their experience at a conference may be a bit overwhelming to society, so we broke this inspirational knowledge fest into a two-parter. We hope that you find something here to inspire you, regardless of gender, to go out into the world and make a difference, even if that difference is solely introspective. We have done our best to carry on the essence of the conference within our tales and more importantly bring along our own experiences in order to offer some sort of guidance if you are here to seek it. Although we do not speak for RVATech or all women, we do speak as six women IN technology from various backgrounds, points in our careers, and stylistic preferences. Take what you want, leave what you don’t and comments or feedback are encouraged!
+
+
+>## Sarah Narum
+>Ippon Engineer | Data | snarum@ipponusa.com
+
+This was my second year attending the RVATech: Women in Technology conference and once again I was blown away by the vivacious, empowering women who led (and attended) this conference. The ever-so-energetic MC, Courtney Ferrell, summarized it best when she said, ‘you need three things to be successful: Confidence, Creative Spirit and Connections’. All of the speakers touched on the importance of these three factors of success in their own way so I’ve decided to address each:
+
+**Confidence** 
+
+Confidence is something I believe everyone struggles with at one (or many) points in their life and career, especially during career changes. Speaking from experience, I was extremely nervous to take the plunge into the world of technology after spending multiple years in the art world, including a Bachelors of Fine Arts. Yes, I studied ceramics in school and now I’ve forged a new path toward software engineering (life is full of surprises). 
+The concept of inclusion was highly talked about, which most often refers to inclusion within a network of people but I believe it applies within oneself as well. I come from a diverse family with my artist father and my programmer mother, so when Avis Yates Rivers said “bring your whole self to whatever you do”, I could not relate more. I’ve found that many times our strengths aren’t always the traditional skill set associated with a job or task. I believe the artist in me allows me to problem solve differently than others on my team and my overall confidence in my creative skills drives me to improve my technical skills. As an art student you put your art, and symbolically yourself, on a pedestal for critiquing which forces you to face your fears of failing. Once again as Rivers stated, you must leave your fears of how you will be interpreted and ‘bring yourself anyways’. 
```

Comment:
Missing blank line between these 2 paragraphs

Reply:
Got it.